scripts/clang-format.sh

```diff
@@ -14,7 +14,7 @@ CPP_DIRS=(
 if which clang-format >/dev/null; then
   DIRS=$(printf "%s " "${CPP_DIRS[@]}")
   find $DIRS -type f \( -name "*.h" -o -name "*.hpp" -o -name "*.cpp" -o -name "*.m" -o -name "*.mm" -o -name "*.c" \) -print0 | while read -d $'\0' file; do
-    clang-format -style=file:./.clang-format -i "$file"
+    clang-format -style=file:.clang-format -i "$file"
```

Comment:
huh? This must be in `config`

Reply:
The problem is, that you cannot configure the GH action to use a different `.clang-format` file path. In the script it would work, but not in the GH workflow

Reply:
https://github.com/jidicula/clang-format-action

Reply:
Ah true. There was a PR to add this tho...

Reply:
fixed the path typo though

Reply:
> Ah true. There was a PR to add this tho...

i'll check it out in 30mins